projects/ion/src/lib/indicator/indicator.component.spec.ts

```diff
@@ -0,0 +1,162 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import {
+  render,
+  RenderResult,
+  screen,
+  fireEvent,
+} from '@testing-library/angular';
+import { ButtonModule } from '../button/button.module';
+import { IonModalComponent } from '../modal/component/modal.component';
+import { TooltipModule } from '../tooltip/tooltip.module';
+import { BodyMockComponent } from './../card/mock/body-mock.component';
+import { IonIndicatorComponent } from './indicator.component';
+import {
+  buttonEmitterConfig,
+  buttonModalConfig,
+  buttonRedirectConfig,
+} from './mocks/indicator-button-config';
+import { IonIndicatorButtonConfiguration } from '../core/types/indicator';
+
+interface IonIndicatorProps {
+  title?: string;
+  tooltipText?: string;
+  value?: string | number;
+  secondValue?: string | number;
+  buttonConfig?: IonIndicatorButtonConfiguration;
+}
+
+@NgModule({
+  entryComponents: [IonModalComponent, BodyMockComponent],
+})
+class EntryComponentModule {}
+
+const sut = async (
+  props?: IonIndicatorProps
+): Promise<RenderResult<IonIndicatorComponent>> => {
+  return await render(IonIndicatorComponent, {
+    imports: [CommonModule, ButtonModule, TooltipModule, EntryComponentModule],
+    declarations: [BodyMockComponent, IonIndicatorComponent, IonModalComponent],
+    componentProperties: props,
+  });
+};
+
+const title = (): HTMLElement => screen.queryByTestId('ion-indicator-title');
```

Comment:
This seems a bit repetitive. You could do an object that maps the test ids for the elements and create a function that receive this object keys.
![image](https://user-images.githubusercontent.com/57760325/217246652-311227fa-13c8-4a32-bff0-e846032a478f.png)